### PR TITLE
Proper Learn Intro Page

### PR DESCRIPTION
**Pull request by CrackTheCode016**
- Similar to `build-index` and `maintain-index`, add a section for learn-index
- Builds a starting point for the `Learn` section of the wiki
- Change all relevant links for `Learn` to start at `learn-index`

### CODE REVIEW

File: docs/learn/learn-index.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+id: learn-index
+title: Learn Polkadot
```
Comment (filippoweb3) [inline docs/learn/learn-index.md]:
```suggestion
title: Learn about Polkadot
```

File: docs/learn/learn-index.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+sidebar_label: Learn Polkadot
+description: Learn how the Polkadot network works, from the ground up.
```
Comment (filippoweb3) [inline docs/learn/learn-index.md]:
```suggestion
description: Learn how the Polkadot network works and what tool you can use to interact with it.
```

File: docs/learn/learn-index.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+keywords: [learn, index, tools, resources]
+slug: ../learn-index
+---
+
+Welcome to the learner's section of the Polkadot Wiki.
+
+Here, you will be able to learn how the Polkadot network works from the ground up - from how
+accounts on Polkadot, to the protocols that power the Polkadot network. If you are looking to begin
+your learning journey, whether you're a developer or someone who is just getting started with
+blockchain.
```
Comment (filippoweb3) [inline docs/learn/learn-index.md]:
```suggestion
Whether you are a developer or someone just getting started with
blockchain, here you will learn how the Polkadot network works from the ground up - from accounts to the protocols that power the Polkadot network.
```

File: docs/learn/learn-index.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+There are a series of video tutorials which are also provided, which you may
+[find here](./learn-video-tutorials.md).
```
Comment (filippoweb3) [inline docs/learn/learn-index.md]:
```suggestion
If you want to learn more about how to use different tools on Polkadot, you can visit the [Polkadot Technical Explainers playlist](https://youtube.com/playlist?list=PLOyWqupZ-WGuAuS00rK-pebTMAOxW41W8) on YouTube.
```

File: docs/learn/learn-index.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+## The Basics: DOT, Accounts, & Staking
+
+Learn the basics of transactions ([extrinsics](./learn-extrinsics)), accounts, staking
+[here](./learn-accounts).
```
Comment (filippoweb3) [inline docs/learn/learn-index.md]:
```suggestion
Learn the basics of transactions, accounts, staking, NFTs, and more.
```

File: docs/learn/learn-index.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+- [NFTs](./learn-nft.md) - Polkadot's approach to NFTs.
+- [Architectural Components](./learn-polkadot-host) - Learn the basics of how the Polkadot
+  architecture works.
+- [Staking](./learn-staking.md) - Learn about how staking works, and how it aids in securing
+  Polkadot.
+
+## Advanced Concepts
+
+Learn more advanced concepts of the Polkadot network, as well as numerous
+[how-to guides for staking](./learn-staking-guides):
```
Comment (filippoweb3) [inline docs/learn/learn-index.md]:
```suggestion
Learn more advanced concepts of the Polkadot network.
```

File: docs/learn/learn-index.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+- [NFT Pallets](./learn-nft-pallets.md) - An overview of the functionalities of NFT Pallets in
+  Polkadot Ecosystem.
+- [Cryptography](./learn-cryptography.md) - Understand what cryptography is used throughout the
+  Polkadot network.
+- [NPoS Algorithms](./learn-phragmen) - Learn about the election method used on Polkadot's Nominated
+  Proof of Staking.
+
+## Network Architecture
+
+Learn more about Polkadot's network architecture in depth:
```
Comment (filippoweb3) [inline docs/learn/learn-index.md]:
```suggestion
Learn more about Polkadot's network architecture in depth.
```

File: docs/learn/learn-index.md [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+- [Parachains](./learn-parachains.md) - An introductory guide to parachains.
+- [Bridges](./learn-bridges.md) - Learn about bridges and common bridge examples.
+- [Cross-Consensus Messaging (XCM)](./learn-xcm.md) - Learn about the messaging format at the
+  forefront of interoperability.
+- [SPREE (Shared Protected Runtime Execution Enclaves)](./learn-spree.md) - Learn about the
+  fundamentals of SPREE.
+
+## Polkadot Comparisons
+
+Learn how Polkadot compares to other similar technologies:
```
Comment (filippoweb3) [inline docs/learn/learn-index.md]:
```suggestion
Learn how Polkadot compares to other similar technologies.
```

File: docs/learn/learn-index.md [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+  Ethereum.
+- [Cosmos](./learn-comparisons-cosmos.md) - Learn about the differences between Polkadot and Cosmos.
+- [Avalanche](./learn-comparisons-avalanche.md) - A high-level comparison between Polkadot and
+  Avalanche.
+- [Other comparisons](./learn-comparisons.md) - Learn how Polkadot compares to Ethereum, Cosmos, and
+  other chains.
+- [Layer Two and Rollups](./learn-comparison-rollups.md) - Comparisons between various rollup and
+  layer two protocols and how they relate to Polkadot.
+
+## Future Implementations & Upcoming Features
```
Comment (filippoweb3) [inline docs/learn/learn-index.md]:
```suggestion
## Future Implementations
```

File: docs/learn/learn-index.md [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+
+Learn upcoming innovations for the Polkadot network that are not yet released:
```
Comment (filippoweb3) [inline docs/learn/learn-index.md]:
```suggestion
Learn about upcoming innovations for the Polkadot network that are not yet implemented.
```

**Review by filippoweb3 — CHANGES_REQUESTED** [review]
@DrW3RK @CrackTheCode016 Added some suggestions. Although the page is beneficial, now that the wiki is under major reorg and sections and sidebar order might change, the page will likely create additional work. I recommend keeping this unmerged, but I also understand if you want to publish it and create PRs whenever the wiki changes.

**Comment by CrackTheCode016** [comment]
@filippoweb3 How much would the reorg it change this page you think? 

If it's just a matter of adding pages / re-ordering a few, I think it could be merged for now. What do you think?

**Review by DrW3RK — APPROVED** [review]
Looks good! Serves as an index page and makes sense to link to this doc when a wiki reader clicks on the "Learn" button. Owing to @filippoweb3 's concern, let us ensure that updates/additions/deletions to the learn section are tracked on this document as well.

**Review by filippoweb3 — APPROVED** [review]

**Pull request merged**